libraries/AP_Logger/AP_Logger.cpp

```diff
@@ -159,8 +159,8 @@ void AP_Logger::Init(const struct LogStructure *structures, uint8_t num_types)
 #endif
 
 #ifdef HAL_LOGGING_DATAFLASH
-    if (_params.backend_types & DATAFLASH_BACKEND_BLOCK) {
-        if (_next_backend == DATAFLASH_MAX_BACKENDS) {
+    if (_params.backend_types &LOGGER_BACKEND_BLOCK) {
```

Comment:
whitespace issue

Reply:
I don't see it now, the rebase must have fixed it